scripts/dev_setup.sh

```diff
@@ -624,6 +624,7 @@ function install_nodejs {
     fi
     install_pkg nodejs "$PACKAGE_MANAGER"
     install_pkg npm "$PACKAGE_MANAGER"
+    curl -fsSL https://yarnpkg.com/install.sh | bash
```

Comment:
I'm not sure I get this.  If developers want to use yarn, they should install it themselves

Reply:
nodejs and NPM do not help the codebase building either? These are tools to help developers start developing and testing the TS code in our codebase. Yarn is an alternative to npm. I remember @wrwg had trouble to figure how to install yarn.